### ISSUE TITLE
download.java.net certificate error - unable to install openjdk

### ISSUE THREAD

**Issue by abdullah-kasim** [se35710/scoop-java#51]
Because download.java.net has a certificate issue, I am unable to proceed with `scoop install openjdk`.

```
PS C:\Users\username> scoop install openjdk
Installing 'openjdk' (12.0.1-12) [64bit]
The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.
URL https://download.java.net/java/GA/jdk12.0.1/69cfe15208a647278a19ef0990eea691/12/GPL/openjdk-12.0.1_windows-x64_bin.zip is not valid
```

**Comment by se35710** [comment]
Even when accepting the faulty certificate, download fails. Need to wait for Oracle to fix the problem.

**Comment by se35710** [comment]
Oracle has fixed the problem. Closes #52 and #53.
